openapi/elmah/client.bal

```diff
@@ -0,0 +1,330 @@
+    # + return - Message where deleted.
+    remote isolated function messagesDelete(string id, string logId) returns http:Response|error {
+        string  path = string `/v3/messages/${logId}/${id}`;
+        map<anydata> queryParam = {api_key: self.apiKeys["api_key"]};
+        path = path + check getPathForQueryParam(queryParam);
+        http:Request request = new;
+        //TODO: Update the request as needed;
+        http:Response response = check self.clientEp-> delete(path, request, targetType = http:Response);
+        return response;
+    }
+    # Fix a message by its ID.
+    #
+    # + id - The ID of the message to fix.
+    # + logId - The ID of the log containing the message.
+    # + markAllAsFixed - If set to true, all instances of the log message are set to fixed.
+    # + return - Message was fixed.
+    remote isolated function messagesFix(string id, string logId, boolean markAllAsFixed = false) returns http:Response|error {
+        string  path = string `/v3/messages/${logId}/${id}/_fix`;
+        map<anydata> queryParam = {"markAllAsFixed": markAllAsFixed, api_key: self.apiKeys["api_key"]};
+        path = path + check getPathForQueryParam(queryParam);
+        http:Request request = new;
+        //TODO: Update the request as needed;
+        http:Response response = check self.clientEp-> post(path, request, targetType = http:Response);
+        return response;
+    }
+    # Hide a message by its ID.
+    #
+    # + id - The ID of the message to hide.
+    # + logId - The ID of the log containing the message.
+    # + return - Message was hidden.
+    remote isolated function messagesHide(string id, string logId) returns http:Response|error {
+        string  path = string `/v3/messages/${logId}/${id}/_hide`;
+        map<anydata> queryParam = {api_key: self.apiKeys["api_key"]};
+        path = path + check getPathForQueryParam(queryParam);
+        http:Request request = new;
+        //TODO: Update the request as needed;
+        http:Response response = check self.clientEp-> post(path, request, targetType = http:Response);
+        return response;
+    }
+    # Fetch a list of uptime checks. Currently in closed beta. Get in contact to get access to this endpoint.
+    #
+    # + return - Request for uptime checks successful.
+    remote isolated function uptimeChecksGetAll() returns UptimeCheck[]|error {
+        string  path = string `/v3/uptimechecks`;
+        map<anydata> queryParam = {api_key: self.apiKeys["api_key"]};
+        path = path + check getPathForQueryParam(queryParam);
+        UptimeCheck[] response = check self.clientEp-> get(path, targetType = UptimeCheckArr);
+        return response;
+    }
+}
+
+# Generate query path with query parameter.
+#
+# + queryParam - Query parameter map
+# + return - Returns generated Path or error at failure of client initialization
+isolated function  getPathForQueryParam(map<anydata>   queryParam)  returns  string|error {
```

Comment:
@aneeshafedo why there is an unwanted space between `map<anydata>` and `queryParam`?

Reply:
This seems like a bug in the code generation.

Reply:
Added an issue https://github.com/ballerina-platform/ballerina-openapi/issues/453